### ISSUE TITLE
Updating to latest version

### ISSUE THREAD

**Issue by ollyboy** [mercator/mercator#9]
Hi all, great project idea. I installed the vagrant box on mac and all went well except for one small error noted in the issue list ( which I have commented on ) 

 Now how do I upgrade after I have done git clone.?

Also would be good to have an example of sending the project to AWS servers:)

**Comment by chnorton** [comment]
Thanks for your interest in Mercator! You can upgrade in much the same way as any other Magento site, which they have helpfully documented:

http://devdocs.magento.com/guides/m1x/install/installing_upgrade_details.html

To update the files for Mercator, you could also update your checked out copy and rebase any changes over the top, or generate a patch from the git changes and apply it.

As for deploying a Mercator project to AWS servers, you can do that just like any other Magento site, using whatever tools you prefer (Phing, Capistrano, Deployer and so forth). The only thing to keep in mind is that the web server configuration will differ from a Magento site due to Mercator using a public directory. You can find more information about this on the [Installing Mercator](https://github.com/mercator/mercator/wiki/Installing-Mercator) page of the wiki.

**Comment by ollyboy** [comment]
Hey I noticed that magento connect doesn't work, is it not possible to install other modules with connect??

Also I like the idea of HHVM but doesn't this mean we can't install modules with ioncube security. I know this is an open source project but was hoping to use it as a base for a medium size retailer

**Comment by chnorton** [comment]
There shouldn't be anything preventing you from installing modules from Magento Connect, although it's possible that something might break due to the public directory split. Could you elaborate on what didn't work?

HHVM is certainly not necessary to run Mercator, which has the same PHP requirements as the equivalent version of Magento. We simply chose to use that as the default interpreter on the Vagrant box. It's perfectly fine to run it on a server which has plain PHP and the Ioncube extension installed.

We detailed how to change the Vagrant box to use `php5-fpm` instead of `hhvm` on the wiki:

https://github.com/mercator/mercator/wiki/Vagrant

**Comment by ollyboy** [comment]
Yep, installed the vagrant box on my macbook air, logged in as admin. If I go to to system>magento connect>magento connection manager I get a 404 error. The VM is definitely connected to internet as I can do apt-get's no probs.

Thanks for the update on HHVM. I know what a slug magento can be and your vagrant box with HHVM runs really fast on the mac and consumes very little resource, I was quite supprised how good it was.

**Comment by ollyboy** [comment]
Hi saw a note that says connect was disabled. Probably a good idea given that all modules should be installed via composer
